moj-node/server/views/components/suggested-content/suggested-content.njk

```diff
@@ -0,0 +1,3 @@
+{% from "suggested-content/macro.njk" import hubSuggestedContent %}
```

Comment:
Delete this one we don't use this file we use the macros.. it's a tech debt to get ride of all of them  😅

Reply:
😆